source/_integrations/github.markdown

```diff
@@ -1,8 +1,9 @@
 ---
 title: GitHub
-description: How to integrate the GitHub sensor into Home Assistant.
+description: How to integrate the GitHub into Home Assistant.
```

Comment:
```suggestion
description: How to monitor GitHub repositories with Home Assistant.
```

At a minimum the word "the" should be removed.  But even then, seems strange to say "integrate github" since it doesn't do commits or pulls or anything.